storybook/stories/SetStateExample.js

```diff
@@ -38,6 +43,7 @@ class SetStateExample extends Component {
           <a href="https://github.com/synapsestudios/draftjs-editor" target="_blank">Github</a>
         </p>
         <DraftJSEditor
+          containerStyle={saneStyling}
```

Comment:
I think what #34 was getting at was that projects shouldn't need to provide these themselves, but rather this library would have sane defaults. In this case we're still relying on the project to pass `saneStyling`, so we should probably add these styles to the CSS for `.DraftJSEditor-root` directly.